transaction-view/src/address_table_lookup_meta.rs

```diff
@@ -103,6 +104,76 @@ impl AddressTableLookupMeta {
     }
 }
 
+pub struct AddressTableLookupIterator<'a> {
+    pub(crate) bytes: &'a [u8],
+    pub(crate) offset: usize,
+    pub(crate) num_address_table_lookup: u8,
+    pub(crate) index: u8,
+}
+
+impl<'a> Iterator for AddressTableLookupIterator<'a> {
+    type Item = SVMMessageAddressTableLookup<'a>;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        if self.index < self.num_address_table_lookup {
```

Comment:
You don't need this if here right? You can iterate based on self.offset. In fact you're accidentally not incrementing self.index now 😅

Reply:
we could but that relies on the fact that `ATLs` are the last items in the packet. Which is true, but would rather not explicitly rely on that for iteration. wdyt?